### DISCUSSION TITLE
Welcome to spotube Discussions!

### DISCUSSION THREAD

**Discussion by KRTirtho** [KRTirtho/spotube#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KRTirtho** [comment]
- Windows: C:\Users\\<user\>\Downloads\Spotube
- Linux: /home/\<user\>/Downloads/Spotube

**Comment by ghost** [comment]
Difficolta' con il login e nessun risultato nei generi musicali...soluzioni?

**Comment by davodbadiei** [comment]
hi, Im using the latest version
And run app and loaded musics
 But my problem is After select the track it stays on the loading track but the track not be load and play
Please help me Thanks 🙏

**Comment by VARA2025** [comment]
BUENAS TARDES amigo como esta por escribirle que su app es de mis favoritas pero hay inconvenientes, la app empezó a tener más fallos:
1. Descargar músicas: la función que descargaba y guardaba en mi almacenamiento para estar sin internet ya no sirve.

2. Reproducción de la música: te reproduce como 15 -20 músicas y se atora.

3. La app se congela y  se crashea mucho: cuando abro la app o quiero reproducir música no vale y me toca estar borrando el almacenamiento.

**Reply by VARA2025** [reply]
Está teniendo muchas fallas amigo la app

**Comment by msadnan007** [comment]
4.0.2 version is dope🔥

**Comment by abc1236089-prog** [comment]
Hi

**Comment by widamlyudha-hub** [comment]
i hope its work 🔥
